packages/apollo-angular/http/tests/http-batch-link.spec.ts

```diff
@@ -36,7 +38,6 @@ describe('HttpBatchLink', () => {
             }
           }
         `,
-        operationName: 'heroes',
```

Comment:
`operationName` is now set by the internal `createOperation` function which is used by `execute`. This guarantees correctness. Setting it externally no longer has any effect